src/Whoops/Handler/PrettyPageHandler.php

```diff
@@ -252,9 +233,12 @@ public function handle()
 
         // Add extra entries list of data tables:
         // @todo: Consolidate addDataTable and addDataTableCallback
-        $extraTables = array_map(function ($table) use ($inspector) {
-            return $table instanceof \Closure ? $table($inspector) : $table;
-        }, $this->getDataTables());
+        $extraTables = array_map(
```

Comment:
I tend to avoid reformatting in the same commit as meaningful changes, as to keep the diffs more meaningful.